### PR TITLE
📚 add readme for blocks package

### LINKED ISSUE #35: Document blocks package
**Is your feature request related to a problem? Please describe.**
As a user, I'd like a guide to understanding why and how I'd use the blocks package.

**Describe the solution you'd like**
See other package readmes.

https://github.com/IBM/slack-wrench/tree/master/packages/blocks

Example: https://github.com/IBM/slack-wrench/blob/master/packages/fixtures/README.md

### PR DESCRIPTION
**Pull request by tteltrab**
**Related Issue**  
closes #35 

**Related PRs**  
This PR is not dependent on any other PR

**What does this PR do?**  
Just adds a readme with an example. probably should have already been there.

**What gif most accurately describes how I feel towards this PR?**

### CODE REVIEW

**Comment by tteltrab** [comment]
Doesn't really _close_ #35, that'll be ongoing, but it's a good enough start to close the issue I think

**Comment by barlock** [comment]
No fair! You would have made me document _every_ helper! 

This looks good to me.

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
   "devDependencies": {
     "@types/jest": "^25.1.0",
-    "@types/node": "^12.12.24",
+    "@types/node": "^13.9.1",
```
Comment (barlock) [inline package.json]: This boy scouting will cause you issues in the rest of your prs 😉

**Comment by tteltrab** [comment]
>No fair! You would have made me document every helper!

Factually untrue because I didn't on accident the last time this was in a PR? lol. Was trying to get a _basic_ version in, definitely can improve over time though.

**Pull request merged**